templates/mail/contact_form_receipt.txt

```diff
@@ -4,22 +4,24 @@ Ce message est un message automatique, merci de ne pas y répondre.
 
 Bonjour,
 
+{% if is_vaccination %}
 Nous vous remercions pour votre message.
 
-{% if is_vaccination %}
-Toutefois, sachez que Acceslibre apporte les informations relatives à l'accessibilité des établissements recevant du public. Nous ne sommes donc pas en charge des modalités de prise de rendez-vous de vaccination.
+Toutefois, sachez que Acceslibre apporte les informations relatives à l'accessibilité des établissements recevant du public.
+Nous ne sommes donc pas en charge des modalités de prise de rendez-vous de vaccination.
 
 Par conséquent, toute demande à destination de l'établissement ne pourra être traitée par nos services.
 
 Nous vous invitons à vous rapprocher de votre centre de vaccination pour répondre à votre demande ou à consulter : https://vaccination-info-service.fr/
+
+Pour toute autre demande, vous recevrez une réponse dans les plus brefs délais.
 {% else %}
-Toutefois, sachez que Acceslibre apporte les informations relatives à l'accessibilité des établissements recevant du public.
+Nous vous remercions pour votre message; vous recevrez une réponse dans les plus brefs délais.
 
-Cependant, la messagerie Acceslibre ne permet pas d'échanger avec les membres des établissements renseignés. Nous vous invitions à contacter directement l'établissement.
+Toutefois, si votre demande s'adresse aux gestionnaires d'établissements, nous vous invitions à contacter directement l'établissement.
```

Comment:
💡 How about mentioning the contact data we have for this ERP? We could mention:

- `contact_email`
- `contact_url`
- `site_internet`
- `telephone`

That way users could contact the ERP directly from this email.